stsagent.py

```diff
@@ -272,7 +272,7 @@ def on_response(self, response):
         if response.error:
             log.error("Response: %s" % response)
             if response.code in RESPONSES_TO_REJECT:
-                self._trManager.tr_error_reject_request(self)
+                self._trManager.tr_error_reject_request(self, response.code)
```

Comment:
"_trManager" formatted not according to PEP8, and of course don't change it if it is coming from upstream like that

Reply:
unfortunately not our code as well.